docs/development.md

```diff
@@ -7,13 +7,16 @@ Refer to [Ktor documentation](https://ktor.io/docs) for detailed building instru
 ## Running locally
 
 ### Configuration
-Create `.env` file in the root of this project and put desired configuration variables in it.
+LebkuchenFM is configured via environment variables.
 Refer to [configuration](./configuration.md) documentation for available configuration variables.
 
+We recommend using `.env` file to store configuration variables for local development.
```

Comment:
maybe this paragraph would be of better use under `Configuration` in docs/configuration.md

Reply:
No, I don't think so. Recommending `.env` files only makes sense for development. When you're deploying you have to adjust to what your server provider is forcing you to use (fly.io admin vs maybe `.env` files when self-hosting).